gateway/1.4/modules/ROOT/pages/flex-local-liveness-check copy.adoc

```diff
@@ -0,0 +1,8 @@
+= Configuring Proxy Protocol in Local Mode
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+:page-mode: local
+
+include::partial$task-config-liveness-probe.adoc[tags=intro1;intro2;k8s-liveness-probe;see-also]
```

Comment:
```suggestion
include::partial$task-config-liveness-probe.adoc[tags=intro1;intro2;see-also]
```
Did you mean to include a liveness probe partial in this doc? Perhaps this is supposed to be a proxy protocol partial?